src/jade/tabs/history.jade

```diff
@@ -194,7 +194,7 @@ section.col-xs-12.content(ng-controller="HistoryCtrl")
                     | Add
                     strong  {{entry.transaction.counterparty | rpcontactname}}
                     |  to contact list
-        nav.paging(ng-if="pagination.count", ng-bind="pagination.pages")
+        nav.paging(ng-if="pagination.count")
```

Comment:
@vhpoet Do you not have to show the page numbers still? Like {{pagination.pages}}?